tests/Artifact.py

```diff
@@ -19,6 +19,14 @@ def testGetArtifactsFromWorkflow(self):
             repr(artifact), 'Artifact(name="vscode-codeql-extension", id=10495898)'
         )
 
+    def testGetArtifactsFromWorkflowWithName(self):
+        artifact = self.repo.get_artifacts(name="vscode-codeql-extension")[0]
```

Comment:
This does not test there is exactly one artifact, it only tests the first returned artifact is as expected.

Reply:
We could always move to a different repo.

Reply:
Assert that we expect multiple artifacts here:
```suggestion
        artifacts = self.repo.get_artifacts(name="vscode-codeql-extension")
        self.assertEqual(artifacts.totalCount, 5420)
        artifact = artifacts[0]
```